libs/designsystem/src/lib/components/modal/modal-wrapper/modal-wrapper.component.ts

```diff
@@ -236,6 +238,19 @@ export class ModalWrapperComponent implements Modal, AfterViewInit, OnInit, OnDe
     }
   }
 
+  private listenForIonModalWillPresent() {
+    if (!this.ionModalElement) return;
```

Comment:
Consider aligning with other methods that checks for `ionModalElement` (i.e. `listenForIonModalDidPresent`, `listenForIonModalWillDismiss`)

Reply:
Fixed in next commit